portfolio/src/main/webapp/script.js

```diff
@@ -12,29 +12,6 @@
 // See the License for the specific language governing permissions and
```

Comment:
Changes in this file are not described in the PR, and they shouldn't be here. I don't know if this is coming from another PR, if not, try to not mix changes in PRs, one PR should be about the same topic.